# shared/exceptions/entity_not_found_exception.py

class EntityNotFoundException(Exception):
    """Exceção lançada quando uma entidade não é encontrada no repositório."""
    pass
